src/blas/backends/cublas/cublas_level1.cpp

```diff
@@ -385,7 +385,7 @@ void sdsdot(sycl::queue &queue, int64_t n, float sb, sycl::buffer<float, 1> &x,
     });
     // Since SB is a host pointer we need to bring the result back to the host and
     // add sb to it.
-    result.get_access<sycl::access::mode::read_write>()[0] += sb;
+    result.get_host_access()[0] += sb;
```

Comment:
I wonder why we don't have (or we have and haven't noticed them yet) these warnings for rocBLAS backend. It has same implementation.

Reply:
LLVM compiler marked it as deprecated recently in January, while hipSYCL (now OpenSYCL) hasn't. I opened [Issue #973](https://github.com/OpenSYCL/OpenSYCL/issues/973) for them.

Reply:
Thanks for checking hipSYCL/openSYCL. I actually meant rocBLAS with LLVM, but as of now, I don't think rocBLAS backend is tested.